apps/admin/frontend/src/screens/manual_data_entry_screen.tsx

```diff
@@ -264,49 +264,58 @@ export function getContestTallyWithUpdatedNumberOfBallots(
 }
 
 export function ManualDataEntryScreen(): JSX.Element {
-  const {
-    electionDefinition,
-    fullElectionManualTally: existingManualData,
-    auth,
-  } = useContext(AppContext);
+  const { electionDefinition, auth } = useContext(AppContext);
   assert(electionDefinition);
   assert(isElectionManagerAuth(auth)); // TODO(auth) check permissions for adding manual tally data
   const { election } = electionDefinition;
-  const { precinctId: currentPrecinctId } =
-    useParams<ManualDataEntryScreenProps>();
+  const {
+    precinctId,
+    ballotStyleId,
+    ballotType: ballotTypeParam,
+  } = useParams<ManualDataEntryScreenProps>();
+  const precinct = find(election.precincts, (p) => p.id === precinctId);
+  const ballotStyle = find(
```

Comment:
We used to have a "could not find this precinct" page but I don't think that really makes sense. We should be, as this is doing implicitly, `assert`ing that the ids in the URL exist. I don't even know how to navigate to any specified URL in `kiosk-browser`.